### PR TITLE
Fix build of new beat after generation

### PR DESCRIPTION
**Pull request by radoondas**
I found out that Created beat is not build-able
This PR  corrects it. 

Please, review. Thanks.

### CODE REVIEW

File: {{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go [hunk 1]
```diff
@@ -70,7 +71,7 @@ func (bt *{{cookiecutter.beat|capitalize}}) Run(b *beat.Beat) error {
 			"type":       b.Name,
 			"counter":    counter,
 		}
-		b.Events.PublishEvent(event)
```
Comment (ruflin) [inline {{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go]: I think this is related to some recent changes in libbeat. Did you update the version used? I think 2 things here are missing:

* Add `b.events = b.Publisher.Connect()` to initialise the events (@urso can provide here more details)
* Update libbeat to the most recent version.

**Comment by radoondas** [comment]
I think it needs review. I have also created an issue regarding new config reading.

**Comment by ruflin** [comment]
@radoondas The Travis build seems to confirm my suspicion. In your version libbeat and the version of beat-generator are not in sync, that is why the build is failing with your changes. But we should definitively update it. See my comments above.

**Comment by radoondas** [comment]
@ruflin sorry, a bit busy at my end :) I believe it is because of lots of changes prior to release 5. Requires to stabilize all development :) And yes, I have not in sync all versions for correct behavior.

**Comment by ruflin** [comment]
@radoondas beat-generator has a fixed libbeat version to make sure every version that is pulled builds. Of course if the libbeat version is updated, changes need to be applied.

**Comment by ruflin** [comment]
Closing this one as the issue will be solved here: https://github.com/elastic/beat-generator/pull/39 Please reopen if there is more things that should be taken care of.

**Pull request closed without merging**